src/Compilers/CSharp/Portable/Symbols/SymbolWithAnnotations.cs

```diff
@@ -182,6 +185,19 @@ public TypeSymbolWithAnnotations AsNullableReferenceOrValueType(CSharpCompilatio
         /// simply returns a symbol for the reference type.
         /// </summary>
         public abstract bool? IsNullable { get; }
+        public abstract bool IsAnnotated { get; }
```

Comment:
It would be good to xml doc this property too. In particular, a nullable value type will have `IsAnnotated==true`, if I followed correctly.